DocumentedExamples/Transport_Through_Straits.ipynb

```diff
@@ -37,8 +37,8 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ureg.define('Sverdrup = 1e6 * meter**3 / second')
</pre><p>I think the Sverdrup is actually defined as <code>sv</code> in pint by default, although the capitalisation is still wrong, I agree! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Can we overwrite `Sv` to be Sverdrup?

Reply:
I don't think so, the closest you can get to overwriting units is to redefine them in a context, but you can't change their dimensionality there. Also seems like a bit too much complexity for a capital letter. Other option is to use a different units definition file, but that's also not an ideal approach. I guess the Sverdrup loses because it's non-SI, anyway...

Reply:
This is the PR where it was added

https://github.com/hgrecco/pint/pull/1404

as a result of this issue

https://github.com/hgrecco/pint/issues/1202

and note [this comment](https://github.com/hgrecco/pint/issues/1202#issuecomment-956625736):

> I think this should be fine (the default registry already has a lot of other domain specific units), but if not you can always add it to cf_xarray.units

So maybe worth asking Deepak Cheerian if this could be overridden by `cf-xarray`?

https://github.com/xarray-contrib/cf-xarray/blob/main/cf_xarray/units.py

Reply:
Thanks @aidanheerdegen. Well when I asked I only meant if we can define a custom unit locally, not really to put it back into `pint` package. I can see why `Sv` would be preferred to be used for `Sievert` which is an SI unit compared to Sverdrup.

I changed the notebook to use the `sverdrup` unit from `pint`. Although it bothers me a bit that it's written with lowercase `sverdrup` but let's live with this. :)